.github/workflows/build.yml

```
@@ -0,0 +1,119 @@
+    if: startsWith(github.ref, 'refs/tags/') # Only run on tag pushes
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Download artifacts
+        uses: actions/download-artifact@v4
+        with:
+          path: artifacts
+
+      - name: Create GitHub Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.4)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)